source/overloading.tex

```diff
@@ -2954,7 +2931,7 @@
 };
 A& operator<<(A&&, char);
 A() << 1;                       // calls \tcode{A::operator<<(int)}
-A() << 'c';                     // calls \tcode{operator<<(A\&\&, char)}
+A() << 'c';                     // calls \tcode{\keyword{operator}<<(A\&\&, char)}
 A a;
 a << 1;                         // calls \tcode{A::operator<<(int)}
 a << 'c';                       // calls \tcode{A::operator<<(int)}
```

Comment:
`\keyword` for all of these comments. (I guess your search missed `operator` after `::`?)